lib/questions.rb

```diff
@@ -1,29 +1,32 @@
 class Questions
-  attr_reader :player_counter
 
-  def initialize()
-    @player_counter = 2
-  end
-
-  def increase_player_counter
-    @player_counter += 1
-  end
+  def turn_count(board)
+    counter = 0
+    board.each do |space|
+       if space == "X" || space == "O"
+          counter += 1
+        end
+    end
+    counter
```

Comment:
There should be a way to do this without having to write a for each. Perhaps using the .count method
https://ruby-doc.org/core-2.7.0/Array.html

Reply:
I'm not sure of the difference in performance.
This is more of a readability issue.

Reply:
10-4